### DISCUSSION TITLE
Acknowledgements for contributors - how to encourage contributons and PhD students

### DISCUSSION THREAD

**Discussion by MansMeg** [swerik-project/the-swedish-parliament-corpus#30] [General SWERIK]
I discussed with Christian Ohm, a researcher at CERN, the problem of giving contributors credits when other researchers contribute to the corpus. We also want to encourage PhD students to work and improve the material.

As I see it, we have two ways to give credits, as authors in papers and acknowledgements. For authorship, we should follow the [Vancouver recommendations](https://www.icmje.org/icmje-recommendations.pdf). Let's try to publish a paper for every major version release of each corpus/repository. And it makes sense that those who contributed since the previous version should be included in the paper. 

The question is, when should someone be invited as a coauthor on a corpus/repo paper? For PhD students, this makes it probably attractive to contribute. Is the equivalent of 120 hours of work large enough to be a co-author? Should all that contributes to previous versions and with minor contributions be acknowledged?

**Comment by fredrik1984** [comment]
Excellent! From the top of my head, 120 hours don't seem too little (three full weeks of work). Do you know what kind of threshold CERN has?

**Reply by MansMeg** [reply]
CERN has a rule that to be an author tou need to work 20% doing mundane, but important, tasks. Roughly.

**Reply by fredrik1984** [reply]
I mean it all depends on what counts as relevant work here I suppose. Maybe, for each paper that we intend to right, we send out an invitation to our Swerik network of people and ask who wants to contribute, and then we can have a discussion of what kind of work, and the scope of it, this would entail et cetera. 20 % seem much. Would it be more pedagogical to say 10 %? That would be about 180 hours.

**Reply by MansMeg** [reply]
This is a discussion. What we can do is open up this possibility. Yes, 20% is a lot. Also, they are the authors of all papers at CERN. So it is different.

I think the important part is to settle on how much work needs to be done (in hours) to be a co-author on the next version. Then we can keep track of those that have made these contributions (given that they also fulfil the Vancouver guidelines) - i.e. it needs to be an intellectual contribution, not just manual annotation done by students. When we decide to release a new version, we will have a list of all the people who have made contributions. 

Maybe a suggestion is then 160h of work is what is expected to be a co-author (ie one month of work)? The rest (and contributors in previous versions) will be mentioned in the acknowledgements?
